scipy/stats/stats.py

```diff
@@ -3441,24 +3441,23 @@ def kendalltau(x, y, initial_lexsort=None, nan_policy='propagate'):
 
     Kendall's tau is a measure of the correspondence between two rankings.
     Values close to 1 indicate strong agreement, values close to -1 indicate
-    strong disagreement.  This is the tau-b version of Kendall's tau which
-    accounts for ties.
+    strong disagreement.  This is the 1945 "tau-b" version of Kendall's
+    tau, which can accounts for ties and which reduces to the 1938 "tau-a"
+    version in absence of ties.
```

Comment:
While you're at it, what would be a good text to refer to for tau-a vs tau-b? Original references are good and should be kept; is there some text which discusses specifically different definitions?